tenants-certified-attributes-updater/src/main/scala/it/pagopa/interop/tenantscertifiedattributesupdater/util/package.scala

```diff
@@ -0,0 +1,131 @@
+package it.pagopa.interop.tenantscertifiedattributesupdater
+
+import akka.actor.typed.ActorSystem
+import cats.implicits._
+import com.typesafe.scalalogging.LoggerTakingImplicit
+import it.pagopa.interop.attributeregistrymanagement.model.persistence.attribute.{Certified, PersistentAttribute}
+import it.pagopa.interop.commons.jwt.service.InteropTokenGenerator
+import it.pagopa.interop.commons.jwt.service.impl.DefaultInteropTokenGenerator
+import it.pagopa.interop.commons.jwt.{JWTInternalTokenConfig, KID, PrivateKeysKidHolder}
+import it.pagopa.interop.commons.logging.ContextFieldsToLog
+import it.pagopa.interop.commons.signer.service.SignerService
+import it.pagopa.interop.partyregistryproxy.client.model.{Institution, Institutions}
+import it.pagopa.interop.tenantmanagement.model.tenant.{PersistentExternalId, PersistentTenant}
+import it.pagopa.interop.tenantprocess.client.model.{ExternalId, InternalAttributeSeed, InternalTenantSeed}
+import it.pagopa.interop.tenantscertifiedattributesupdater.system.ApplicationConfiguration
+import org.mongodb.scala.MongoClient
+
+import java.util.UUID
+import scala.concurrent.{ExecutionContext, Future}
+
+package object util {
+  def generateBearer(jwtConfig: JWTInternalTokenConfig, signerService: SignerService)(implicit
+    actorSystem: ActorSystem[_],
+    ec: ExecutionContext
+  ): Future[String] = for {
+    tokenGenerator <- interopTokenGenerator(signerService)
+    m2mToken       <- tokenGenerator
+      .generateInternalToken(
+        subject = jwtConfig.subject,
+        audience = jwtConfig.audience.toList,
+        tokenIssuer = jwtConfig.issuer,
+        secondsDuration = jwtConfig.durationInSeconds
+      )
+  } yield m2mToken.serialized
+
+  private def interopTokenGenerator(
+    signerService: SignerService
+  )(implicit actorSystem: ActorSystem[_], ec: ExecutionContext): Future[InteropTokenGenerator] =
+    Future(
```

Comment:
Building it with `Future.successful` will prevent the `ec` from being required in the method probably, but it's a refactor not worth a commit.